### PR TITLE
refactor(application): ensure cloudProviders is a comma-separated string

### PR DESCRIPTION
**Pull request by anotherchrisberry**
https://github.com/spinnaker/orca/pull/2604 assumes `cloudProviders` is a string, so let's make sure it is

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#Thu Jan 24 12:23:44 PST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
```
Comment (ajordens) [inline gradle/wrapper/gradle-wrapper.properties]: why did you change this

Comment (anotherchrisberry) [reply gradle/wrapper/gradle-wrapper.properties]: wut

**Comment by anotherchrisberry** [comment]
I will do as a migrator instead

**Comment by anotherchrisberry** [comment]
I will do as a migrator, and I will do as a this change.
